### ISSUE TITLE
(Accessibility) Signal is not being called in the template

### ISSUE THREAD

**Issue by tom-vism** [ng-select/ng-select#2698]
### Describe the bug

When no items are found, the screen reader should announce the not found text; however, the signal property is not being called, therefore the screen reader reads the definition and not the actual value.

### Steps to Reproduce

1. Open the demo page
2. Launch the screen reader, 
3. Type random letters in the searchable field so that nothing is found.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://ng-select.github.io/ng-select/#/fixed-placeholder

### Expected behaviour

Expected: Screen reader reads the not found text
Actual: read the definition and not the actual value

### What version of the library you see this issue?

20.2.2

### What version of Angular are you using?

Angular 20

### Screenshots

<img width="606" height="125" alt="Image" src="https://github.com/user-attachments/assets/ea0bb017-1b2a-4f43-aadf-012977dd7c57" />
